# import_test

# Purpose
I originally had a problem with using the PyCharm debugger on a complex Python script.  I simplified this down to a simple main script and a simple custom module and opened [Can run simple app with custom import in PyCharm but debugger can't resolve import](https://intellij-support.jetbrains.com/hc/en-us/community/posts/360004173059-Can-run-simple-app-with-custom-import-in-PyCharm-but-debugger-can-t-resolve-import) in the PyCharm community and created this repo with the scripts to demonstrate the issue.
